### PR TITLE
added the landing page

### PR DESCRIPTION
**Pull request by ShivaKakarla512**
Styles and assets may change

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -25,7 +25,8 @@
     ],
     "rules": {
         "indent": ["error", "tab"],
-        "no-tabs": "off"
+        "no-tabs": "off",
```
Comment (dhee9000) [inline .eslintrc:28]: drop eslintrc changes, use Unix line endings

File: src/views/viewLandingPage/ViewLandingPage.css [hunk 2]
```diff
@@ -0,0 +1,156 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&display=swap");
```
Comment (dhee9000) [inline src/views/viewLandingPage/ViewLandingPage.css:1]: will get on call with you about this file

File: src/views/viewLandingPage/ViewLandingPage.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import './ViewLandingPage.css';
+// import { useAuth0 } from '@auth0/auth0-react';
+import Vinattalogo from './assets/VinattaLogo.png';
+import pic from './assets/doctorthree.svg';
```
Comment (dhee9000) [inline src/views/viewLandingPage/ViewLandingPage.tsx:5]: let's use a more descriptive name

File: src/views/viewLandingPage/ViewLandingPage.tsx [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+							<br />
+							<span>Vinatta</span>
+						</h1>
+						<p className="landing_description">
+							Health Equity, Always.
+						</p>
+						<a
+							href="#"
+							className="landing_button"
+							// onClick={() => loginWithRedirect()}
```
Comment (dhee9000) [inline src/views/viewLandingPage/ViewLandingPage.tsx:58]: is this going to be uncommented in the future? if not let's remove commented lines

**Review by dhee9000 — CHANGES_REQUESTED** [review]
Great work on this, left some feedback but overall almost ready to approve.

**Review by MatthewKlayme — APPROVED** [review]
looks good

**Review by dhee9000 — APPROVED** [review]
Looks great! Ready to merge in to develop

**Pull request merged**